app/controllers/developers/applications_controller.rb

```diff
@@ -1,20 +1,18 @@
 class Developers::ApplicationsController < ApplicationController
-  nested_layouts "layouts/admin"
+  nested_layouts 'layouts/admin'
 
   before_action do
     @developers = true
   end
 
-  before_action except: [:index, :request, :provision, :create] do
+  before_action except: %i[index request provision create] do
```

Comment:
what's the rationale behind this change, feels less descriptive imo?

Reply:
uhhhh the weird formatter did that